src/main/java/core/basesyntax/service/RegistrationServiceImpl.java

```diff
@@ -2,13 +2,37 @@
 
 import core.basesyntax.dao.StorageDao;
 import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.exception.ValidationException;
 import core.basesyntax.model.User;
 
 public class RegistrationServiceImpl implements RegistrationService {
-    private final StorageDao storageDao = new StorageDaoImpl();
+    private static final int MIN_LENGHT = 6;
+    private static final int USER_MIN_AGE = 18;
+    private StorageDao storageDao;
 
     @Override
     public User register(User user) {
-        return null;
+        storageDao = new StorageDaoImpl();
```

Comment:
you are creating new storageDao each time the method is called. Let's initialize it in constructor or directly in field